seed/static/seed/js/controllers/data_quality_admin_controller.js

```diff
@@ -223,6 +258,16 @@ angular.module('BE.seed.controller.data_quality_admin', [])
                   r.label = match.id;
                 }
               }
+              if(!(r.min === '' || r.min === null) && !(r.max === '' || r.max === null)) {
+                if (r.max < r.min) {
+                  var min = r.min;
+                  r.min = r.max;
+                  r.max = min;
+                }
```

Comment:
Nice! I like this feature but would generally suggest that small, unrelated cleanups be done in a separate PR when the current PR is as large as this one - unless, of course, somebody specifically requested it be done in this one 😄